src/controllers/room.controller.js

```diff
@@ -0,0 +1,82 @@
+import { Room } from '../models/Room.models.js';
+
+export const getRooms = async (req, res) => {
+  res.send(await Room.findAll());
+};
+
+export const getOneRoom = async (req, res) => {
+  const { id } = req.params;
+  const room = await Room.findByPk(id);
+
+  if (!room) {
+    res.status(404).send({ message: 'cant get a room' });
+
+    return;
+  }
+  res.status(200);
+  res.send(room);
+};
+
+export const createRoom = async (req, res) => {
+  const { roomName } = req.body;
+
+  if (!roomName) {
+    res.status(400).send({ message: 'room name is required' });
+  }
+
+  const room = {
+    roomName,
+  };
+
+  const newRoom = await Room.create(room);
+
+  res.status(201);
```

Comment:
In your `createRoom` function, you are setting the status code to 201 but you are not ending the response. This could lead to hanging requests as the response is never closed. You should send the response in the same line you set the status code. It would look like this: `res.status(201).send(newRoom);`